### ISSUE TITLE
Settings not availabe

### ISSUE THREAD

**Issue by fellmann** [palantir/eclipse-typescript#352]
I am using the TypeScript editor to edit JSX files inside a Java (Spring Boot) project in Eclipse.
Using decorators, I get the warning:
```Experimental support for decorators is a feature that is subject to change in a future release. Set the 'experimentalDecorators' option to remove this warning.```
Yet, there is no TypeScript settings page in the project settings or any other place I can find.

**Comment by Rouche** [comment]
![image](https://cloud.githubusercontent.com/assets/1267438/25230279/6115980a-25a1-11e7-847b-4e54683b1be1.png)

**Comment by fellmann** [comment]
![image](https://cloud.githubusercontent.com/assets/10139607/25232010/39adb05c-25da-11e7-9fcb-58581aa82381.png)

There is no TypeScript section in my preferences.

**Comment by Rouche** [comment]
Weird, this seems to be another plugin. Not Palantir's. Or maybe an older version?

I dont have this option under Javascript.

**Comment by diogosantana** [comment]
This is [TypeScript IDE typescript.java](https://github.com/angelozerr/typescript.java) plugin.
